examples/panel-datalinks/README.md

```diff
@@ -10,6 +10,8 @@ Use panel plugins when you want to do things like visualize data returned by dat
 
 ## Get started
 
+A data source backend plugin consists of both frontend and backend components. Install these with the following CLI commands.
```

Comment:
Again backend is optional and this one doesnt have a backend

Reply:
done

Reply:
I don't understand the context of this sentence. This is plugin is a panel, why is a data source mentioned here? why in the "getting started" section?